docs/contributing.md

```diff
@@ -43,14 +43,31 @@ you should still test your changes.
 
 This means 3 things
 
-* Compiling Catch's SelfTest project -- code that does not compile is evidently incorrect. Obviously, you are not expected to
+* Compiling Catch's SelfTest project:
+```
+$ cd Catch2
+$ cmake -Bcmake-build-debug -H. -DCATCH_BUILD_TESTING=ON
+$ cd cmake-build-debug/
+$ make
+```
+-- code that does not compile is evidently incorrect. Obviously, you are not expected to
 have access to all compilers and platforms Catch supports, Catch's CI pipeline will compile your code using supported compilers
 once you open a PR.
-* Running the SelfTest binary. There should be no unexpected failures on simple run.
-* Running Catch's approval tests. Approval tests compare current output of the SelfTest binary in various configurations against
-known good output. Catch's repository provides utility scripts `approvalTests.py` to help you with this. It needs to be passed
-the SelfTest binary compiled with your changes, like so: `$ ./scripts/approvalTests.py clang-build/SelfTest`. The output should
-be fairly self-explanatory.
+
+* Running the SelfTest binary:
+```
+$ ./cmake-build-debug/projects/SelfTest
```

Comment:
just a note that on Windows the binary is one more folder deeper usually Debug

Reply:
Oh, good to know! I added a note for that, I hope I got the path right.